multires_consensus_clustering/informativ_plots.py

```diff
@@ -96,7 +97,11 @@ def cell_occurrence_plot(graph, adata, clustering_data):
 
     # plot cell counts with scanpy and umap
     adata.obs["cell_counts"] = cell_counts_df["cell_counts"]
-    plot = sc.pl.umap(adata, color=["cell_counts"], show=True)
+    if plot:
+        plt.tight_layout()
+        sc.pl.umap(adata, color=["cell_counts"], show=True)
+
+    return cell_counts_df["cell_counts"].values
```

Comment:
Maybe a return argument is better than `plot`? At least from the name of the function I would expect a plot to be the default output. Or maybe this should be two functions?